doctor/views.py

```diff
@@ -182,14 +182,17 @@ def xray(request) -> JsonResponse:
             {"error": True, "msg": "Failed validation"}, status=BAD_REQUEST
         )
     extension = form.cleaned_data["extension"]
-    if "pdf".casefold() != extension.casefold():
+    if "pdf" != extension.casefold():
         return JsonResponse(
             {"error": True, "msg": "Failed file type"}, status=BAD_REQUEST
         )
     results = get_xray(form.cleaned_data["fp"])
     if results.get("error", False):
         return JsonResponse(results, status=BAD_REQUEST)
-    cleanup_form(form)
+    try:
+        cleanup_form(form)
+    except FileNotFoundError:
+        pass
```

Comment:
Sorry, this wasn't what I meant. What I meant was that if any part of the above view fails, this cleanup_form won't get run, and if it doesn't get run, then the file will be left lingering around forever on disk.

So I think the fix is probably something like:

```
def xray(request) -> JsonResponse:
    try:
        ...
    except:
        pass
    finally:
        cleanup_form(form)
    return ...
```

This also feels a lot like a decorator that cleans up on exit of the decorated function, but I haven't thought too hard about that.